src/is/arithmetic.js

```diff
@@ -0,0 +1,24 @@
+import types from './types';
+
+/**
+ * Returns whether a given value is even
+ * @param {*} value the value to check
+ */
+export function isEven(value) {
+    if (!types.number(value)) return false;
+    return value % 2 === 0;
+}
+
+/**
+ * Returns whether a given value is odd
+ * @param {*} value the value to check
+ */
+export function isOdd(value) {
+    if (!types.number(value)) return false;
+    return !isEven(value);
```

Comment:
Function call adds an unnecessary check (in `isEven`). You should directly check the value.